addons/l10n_pe_website_sale/views/templates.xml

```diff
@@ -0,0 +1,107 @@
+            </t>
+        </div>
+
+    </template>
+
+    <template id="partner_address_info" name="Peruvian partner address">
+
+        <!-- show city -->
+        <div t-attf-class="mb-3 #{error.get('city_id') and 'o_has_error' or ''} col-lg-6 div_city_id"
+            t-attf-style="#{(country and country.code != 'PE') and 'display: none' or ''}">
+            <label class="col-form-label" for="city_id">City</label>
+            <select id="city_id"
+                name="city_id"
+                t-attf-class="form-select #{error.get('city_id') and 'is-invalid' or ''}"
+                data-init="1">
+                <t t-foreach="state_cities" t-as="city">
+                    <option t-att-value="city.id"
+                        t-att-selected="city.id == ('city_id' in checkout and checkout['city_id'] != '' and int(checkout['city_id']))">
+                        <t t-out="city.name" />
+                    </option>
+                </t>
+            </select>
+        </div>
+
+        <!-- show district -->
+        <div t-attf-class="mb-3 #{error.get('l10n_pe_district') and 'o_has_error' or ''} col-lg-6 div_district"
+            t-att-style="not city and 'display: none'">
+            <label class="col-form-label" for="l10n_pe_district">District</label>
+            <select id="l10n_pe_district"
+                name="l10n_pe_district"
+                t-attf-class="form-select #{error.get('l10n_pe_district') and 'is-invalid' or ''}"
+                data-init="1">
+                <t t-foreach="city_districts" t-as="district">
+                    <option t-att-value="district.id"
+                        t-att-selected="district.id == ('l10n_pe_district' in checkout and checkout['l10n_pe_district'] != '' and int(checkout['l10n_pe_district']))">
+                        <t t-out="district.name" />
+                    </option>
+                </t>
+            </select>
+        </div>
+
+    </template>
+
+    <template id="address" inherit_id="website_sale.address">
+        <xpath expr="//input[@name='vat']/.." position="before">
+            <t t-if="mode[1] == 'billing'" position="inside">
+                <t t-if="res_company.country_id.code == 'PE'">
+                    <t t-set="partner" t-value="website_sale_order.partner_id" />
+                    <t t-call="l10n_pe_website_sale.partner_info" />
+                </t>
+            </t>
+        </xpath>
+        <label for="vat" position="attributes">
+            <attribute name="t-if">res_company.country_id.code != 'PE'</attribute>
+        </label>
+        <label for="vat" position="after">
+            <label t-if="res_company.country_id.code == 'PE'" class="col-form-label label-optional" for="vat">
+                Number
+            </label>
+        </label>
+        <xpath expr="//select[@name='country_id']" position="attributes">
+            <attribute name="t-att-data-company_country_code">res_company.country_id.code</attribute>
```

Comment:
How is this override PE-specific?

Reply:
Is an extra attribute that is used [here](https://github.com/odoo/odoo/pull/130155/files#diff-3e73cdaf8077b89c48579b44b997cff31cf049b58fc66ed417209e307f5b0de4R16)